sdk/textanalytics/ai-text-analytics/src/analyzeSentimentResult.ts

```diff
@@ -64,6 +64,14 @@ export interface SentenceSentiment {
    * The sentiment confidence score between 0 and 1 for the sentence for all classes.
    */
   confidenceScores: SentimentConfidenceScores;
+  /**
+   * The sentence offset from the start of the document.
```

Comment:
It will be better to say "The sentence text offset..."

Reply:
Also could you add to all the docstrings for length and offset that the encoding for is Utf16CodeUnit